federated-learning-controller/docs/why-fl-in-ocm.md

```diff
@@ -0,0 +1,29 @@
+# Why Federated Learning Fits Open Cluster Management
```

Comment:
Summary - They Both follow a hub-spoke arch, with pull-based model:
- In OCM, managed clusters pull their desired state and push status back to the hub.
- In FL, collaborators pull the global model, train locally, then push model updates back to the aggregator.
- the OCM Hub acts like the aggregator or server,
And each Managed Cluster plays the role of collaborator or client in FL.